### PR TITLE
UI clean up 

### PR DESCRIPTION
**Pull request by kkateq**
- removal of dead components 
- removed stubbed notifications list from HOME page and replaced with My pull requests
- few other changes

### CODE REVIEW

File: src/client/pages/Project/PullRequest/ActionBar/index.js [hunk 1]
```diff
@@ -1,117 +1,70 @@
...
-                    </div>
-                    <div style={{ margin: '5px 15px' }}>Guardian persona</div>
-                  </div>
-                </MenuItem>
-              </DropdownButton>
-            </ButtonGroup>
-          </Col>
-        </Row>
+        <ButtonGroup style={{ float: 'right' }}>
+          <Button style={approveButtonStyle} onClick={this.handleApprovePullRequest}>
```
Comment (mariangemarcano) [inline src/client/pages/Project/PullRequest/ActionBar/index.js]: @kkateq I was thinking we could use the Approved/Request changes component that I built so the users has a quick hand of the 2 options at hand.

File: src/client/pages/Project/PullRequest/Layouts/LayoutDeveloper.js [hunk 2]
```diff
@@ -7,10 +7,9 @@ import CategoryModule from './common'
 
 const CATEGORIES = [
   { url: 'summary', name: 'Summary' },
-  { url: 'discussion', name: 'Discussion' },
+  { url: 'diff', name: 'Files' },
```
Comment (mariangemarcano) [inline src/client/pages/Project/PullRequest/Layouts/LayoutDeveloper.js]: We should probably call it diff?

File: src/client/pages/Project/PullRequest/ActionBar/index.js (hunk 1 above)
Comment (kkateq) [inline src/client/pages/Project/PullRequest/ActionBar/index.js]: yes, sure, feel free to replace, I didn't want to interfere with your changes, so just removed nonsense from there and placed possible mock of what we are expecting to complete.

File: src/client/pages/Project/PullRequest/Layouts/LayoutDeveloper.js (hunk 2 above)
Comment (kkateq) [inline src/client/pages/Project/PullRequest/Layouts/LayoutDeveloper.js]: I intentionally renamed to Files, don't like `diff` as it sort of incomplete, maybe `Files diff`, or just `Files` :)

File: src/client/pages/Project/PullRequest/ActionBar/index.js (hunk 1 above)
Comment (mariangemarcano) [inline src/client/pages/Project/PullRequest/ActionBar/index.js]: @kkateq ok cool I'll do it afterwards.

File: src/client/pages/Project/PullRequest/Layouts/LayoutDeveloper.js (hunk 2 above)
Comment (mariangemarcano) [inline src/client/pages/Project/PullRequest/Layouts/LayoutDeveloper.js]: Ok @kkateq I would suggest File diff or Files changed.

Comment (kkateq) [inline src/client/pages/Project/PullRequest/Layouts/LayoutDeveloper.js]: Done.

**Pull request merged**